packages/cli/src/commands/process/logs.ts

```diff
@@ -61,6 +61,24 @@ export default class ProcessLogs extends Command {
       .on('data', this.handlerResult(process.hash))
       .on('error', (error: Error) => { this.warn('Result stream error: ' + error.message) })
 
+    const engines = await this.listServices({
+      name: flags.name
+    });
+    if (engines.length === 0) {
+      throw new Error("No engine is running.")
+    }
+    const engine = engines[0];
+    const logs: any = await engine.logs({
+      stderr: true,
+      stdout: true
+    })
+    logs.on('data', (buffer: Buffer) => parseLog(buffer).forEach(x => {
+      if (x.includes("module=orchestrator")) this.log(x)
+    }))
+    logs.on('error', (error: Error) => {
+      throw error
+    })
+    
     return {
       destroy: () => {
         streams.forEach(s => s())
```

Comment:
This `streams` variable is supposed to contain all the functions to destroy the streams, we should add the stream of the engine to make sure we stop it at the end

Reply:
I'm not sure how to do this, the `logs` are merely string buffers, is there a way to add a cancellation function for `logs` to `streams`?